### PR TITLE
Feature/handler size gg23 gg24

### PR DESCRIPTION
**Pull request by tadeuif**
Size ausente e size inválido

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -3,3 +3,5 @@
 
 def run():
     execute()
+
+run()
```
Comment (lucasfernando) [inline main.py]: Why is this line here?

File: src/generators/handler.py [hunk 2]
```diff
@@ -21,7 +21,7 @@ def get_valid_specification(self):
         config = config_reader.get_config()
 
         if 'datasets' not in config:
-            raise ValueError("Malformed specification file")
+            raise ValueError("Não possui datasets")
```
Comment (lucasfernando) [inline src/generators/handler.py]: All the messages by default are in English

File: src/generators/handler.py [hunk 3]
```diff
@@ -32,7 +32,11 @@ def get_valid_specification(self):
                 format=datasets[key]['format'],
                 serializers=datasets[key]['serializers']
             )
-            if dataset_validator.is_valid() is False:
+            if dataset_validator.size <= 0:
+                raise ValueError("Propriedade ‘size’ deve ser maior que 0 (zero)")
```
Comment (lucasfernando) [inline src/generators/handler.py]: English

File: src/generators/handler.py [hunk 4]
```diff
@@ -32,7 +32,11 @@ def get_valid_specification(self):
...
+            if dataset_validator.size == "":
+                '''raise ValueError("Dataset XPTO: Propriedade ‘size’ é obrigatória")''' #levantando a exceção de arquivo mal formado no validator.py
```
Comment (lucasfernando) [inline src/generators/handler.py]: I think this line is unnecessary

File: src/lib/config/validator.py [hunk 5]
```diff
@@ -9,11 +9,14 @@ def __init__(self, file_path):
...
-        file.close()
-        return configuration
 
+        try:
+            file = open(self.file_path, 'r')
+            configuration = json.loads(file.read())
+            file.close()
+            return configuration
+        except ValueError:
+            raise ValueError("Arquivo mal formado/campo vazio")
```
Comment (lucasfernando) [inline src/lib/config/validator.py]: English

File: src/lib/config/validator.py [hunk 6]
```diff
@@ -9,11 +9,14 @@ def __init__(self, file_path):
         self.file_path = file_path
 
     def get_config(self):
-        file = open(self.file_path, 'r')
-        configuration = json.loads(file.read())
-        file.close()
-        return configuration
 
+        try:
```
Comment (lucasfernando) [inline src/lib/config/validator.py:17]: What do you think about use try catch just around the `json.loads`

**Review by lucasfernando — CHANGES_REQUESTED** [review]
There are a fews things to fix and please remove fake_generator_sample1

**Comment by tadeuif** [comment]
Guys, now when we have the size error, the program will raise a more friendly message. Other changes are done with the #35 PR. Can you pls review it?
Tks!

**Review by lucasfernando — APPROVED** [review]
We can do some improvements, but I think this should work for now.

**Pull request merged**